manifests/x/XunLei/xunlei11/11.2.4.1750/XunLei.xunlei11.yaml

```
@@ -0,0 +1,5 @@
+PackageIdentifier: "XunLei.xunlei11"
```

```suggestion
PackageIdentifier: "XunLei.xunlei"
```
Oops, I forgot this.